utilities/installer/install_puppet.rb

```diff
@@ -122,6 +122,9 @@
 #
 ################################################################################
 
+require 'beaker-pe'
```

Comment:
Just curious.  Did you test this installer?  We added a deprecation notice since we no longer technically support it but since you made changes to it it sparked my interest.

Reply:
Ah I wasn't aware that it had been deprecated, but this was the installer that I have been using to put an agent on our VMs for testing dual mode.

Reply:
Good to know it still works :-) . We only deprecated it because we were informed that Beaker has a native installer that can be used to install the agent.  Is that still the case?

Reply:
FYI, we added the deprecation notice in the doc that describes the installer workflow.

https://github.com/cisco/cisco-network-puppet-module/blob/develop/docs/README-beaker-agent-install.md

Reply:
> Good to know it still works :-) . We only deprecated it because we were informed that Beaker has a native installer that can be used to install the agent. Is that still the case?

https://docs.puppet.com/pe/2015.3/install_nxos.html#step-2-perform-a-bash-install-or-a-guest-shell-install is the docs I can see for installing the agent, installing it via Beaker should be possible but I am not sure if it has been documented/tested - would need to investigate further.